pkg/operator/controller/ingress/controller_test.go

```diff
@@ -317,3 +317,60 @@ func TestTLSProfileSpecForIngressController(t *testing.T) {
 		}
 	}
 }
+
+func TestValidateHTTPHeaderBufferValues(t *testing.T) {
+	testCases := []struct {
+		description      string
+		httpHeaderBuffer operatorv1.IngressControllerHTTPHeaderBuffer
+		valid            bool
+	}{
+		{
+			description:      "empty httpHeaderBuffer",
+			httpHeaderBuffer: operatorv1.IngressControllerHTTPHeaderBuffer{},
+			valid:            true,
+		},
+		{
+			description: "valid, non-empty httpHeaderBuffer",
+			httpHeaderBuffer: operatorv1.IngressControllerHTTPHeaderBuffer{
+				HeaderBufferBytes:           32768,
+				HeaderBufferMaxRewriteBytes: 8192,
+			},
+			valid: true,
+		},
+		{
+			description: "invalid, non-empty httpHeaderBuffer",
+			httpHeaderBuffer: operatorv1.IngressControllerHTTPHeaderBuffer{
+				HeaderBufferBytes:           8192,
+				HeaderBufferMaxRewriteBytes: 32768,
+			},
+			valid: false,
+		},
+		{
+			description: "invalid httpHeaderBuffer, HeaderBufferMaxRewriteBytes empty",
```

Comment:
nit: empty =~ "not specified"? Just trying to distinguish whether empty =~ the zero value.

Reply:
should be fixed now.